KsApi/Service+RequestHelpers.swift

```diff
@@ -41,6 +41,26 @@ extension Service {
     ).handle_combine_dataResponse(service: self)
   }
 
+  func requestWithAuthentication<Model: Decodable>(_ route: Route,
```

Comment:
The amount of copy-paste here makes me unhappy. However, I poked around with a few ideas for refactoring, and realized it was going to take significantly _more_ refactoring to make a dent in this. I decided that (for the moment) this was an OK evil. 

Because this is only used by precisely one route, I would much rather have one ugly copy-pasted method than add an extra "authentication header" parameter throughout our whole networking stack. And I would rather ship this now than embark in a larger refactoring project for `Route`.

Reply:
Just to confirm: this is necessary because we're calling this as soon as we have the token; before the token has been set (since we set the token when we have the user as well)? If I'm understanding this correctly, I support this as an OK evil, especially if we still think we may get the user object returned with the token in the future.